pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -230,6 +257,14 @@ func (bd *backupDriver) monitorBackups(ctx context.Context) {
 		backupCountQuery := fmt.Sprintf(`SELECT count(DISTINCT end_time) FROM [SHOW BACKUP FROM LATEST IN '%s']`, fixtureURI.String())
 		sql.QueryRow(bd.t, backupCountQuery).Scan(&backupCount)
 		bd.t.L().Printf(`%d scheduled backups taken`, backupCount)
+
+		compSuccess, compRunning, compFailed := bd.compactionJobStates(sql)
+		if bd.sp.fixture.CompactionThreshold > 0 {
```

Comment:
Currently, this logs the compacted jobs but doesn't verify that compaction completes successfully. What do you think about having monitorBackups wait for compactions to finish and fail the test if any compaction errors?

Reply:
Yeah sure, that sounds good to me